src/mlpack/core/dists/gamma_distribution.cpp

```diff
@@ -0,0 +1,76 @@
+/**
+ * @file gamma_distribution.cpp
+ * @author Yannis Mentekidis
+ *
+ * Implementation of the methods of GammaDistribution.
+ */
+#include "gamma_distribution.hpp"
+
+#include <boost/math/special_functions/digamma.hpp>
+#include <boost/math/special_functions/trigamma.hpp>
```

Comment:
I replaced trigamma(.) with polygamma(2, .) to see if that works. According to [this](http://mathworld.wolfram.com/PolygammaFunction.html), I could avoid including it completely and compute it using digamma. I'm trying that while travis attempts to build the new version.